concepticondata/conceptlists/Rowe-1954-232.tsv

```diff
@@ -0,0 +1,233 @@
+Rowe-1954-232-103	103	murciélago	bat	1793	BAT
+Rowe-1954-232-104	104	nutria	otter	15	OTTER
+Rowe-1954-232-105	105	capuchin, machín	cebus monkey	439	CEBUS MONKEY
+Rowe-1954-232-106	106	mono aullador, coto, araguato	howler monkey	433	HOWLER MONKEY
+Rowe-1954-232-107	107	marimonda, maquisapa	spider monkey	434	SPIDER MONKEY
+Rowe-1954-232-108	108	ronsoco, carpincho, chigüire	capybara	314	CAPYBARA
+Rowe-1954-232-109	109	oso hormiguero, melero, palmero	anteater	181	ANTEATER
+Rowe-1954-232-110	110	paca, guagua, majás, lapa	paca	316	PACA
+Rowe-1954-232-111	111	caimán, lagarto	crocodile	1857	CROCODILE
+Rowe-1954-232-112	112	iguana	iguana	1578	IGUANA
+Rowe-1954-232-113	113	tortuga de tierra, morrocoy	boxturtle	895	TORTOISE
+Rowe-1954-232-114	114	tortuga de agua, galápago, motelo	river turtle	112	TURTLE
+Rowe-1954-232-115	115	saino, sajino, báquiro de collar, chácharo	collared peccary	435	COLLARED PECCARY
+Rowe-1954-232-116	116	jabalí mayor, guangana, báquiro	white-lipped peccary	436	WHITE-LIPPED PECCARY
+Rowe-1954-232-117	117	agutí, acure	agouti	313	AGOUTI
+Rowe-1954-232-118	118	gato	cat	1208	CAT
+Rowe-1954-232-119	119	rata	rat	1490	RAT
+Rowe-1954-232-120	120	ratón	mouse	1491	MOUSE
+Rowe-1954-232-121	121	cola, rabo	tail	1220	TAIL
+Rowe-1954-232-122	122	serpiente, culebra	snake	730	SNAKE
+Rowe-1954-232-123	123	anaconda, culebra de agua	anaconda, water boa	896	ANACONDA (WATER BOA)
+Rowe-1954-232-124	124	cascabel	rattlesnake	871	RATTLESNAKE
+Rowe-1954-232-125	125	coral	coral snake	2240	CORAL SNAKE
+Rowe-1954-232-126	126	sapo	toad	894	TOAD
+Rowe-1954-232-127	127	pájaro	bird	937	BIRD
+Rowe-1954-232-128	128	picaflor, colibrí, tucusito	humming bird	1589	HUMMINGBIRD
+Rowe-1954-232-129	129	guacamayo	macaw	1630	MACAW
+Rowe-1954-232-130	130	tucán, pinsha	toucan	308	TOUCAN
+Rowe-1954-232-131	131	loro	parrot	882	PARROT
+Rowe-1954-232-132	132	perico, pihuicho	parakeet	886	PARAKEET
+Rowe-1954-232-133	133	gallinazo, zamuro	buzzard	1865	BUZZARD
+Rowe-1954-232-134	134	pavo, guaco, paují	curassow	315	CURASSOW
+Rowe-1954-232-135	135	lechuza	owl	735	OWL
+Rowe-1954-232-136	136	paya de monte	guan	437	GUAN
+Rowe-1954-232-137	137	gallina	hen	1514	HEN
+Rowe-1954-232-138	138	pescado	fish	227	FISH
+Rowe-1954-232-139	139	caribe, palometa	piranha	317	PIRANHA
+Rowe-1954-232-140	140	abeja	bee	665	BEE
+Rowe-1954-232-141	141	mosca	fly	1504	FLY (INSECT)
+Rowe-1954-232-142	142	pulga	flea	232	FLEA
+Rowe-1954-232-143	143	piojo	louse	1392	LOUSE
+Rowe-1954-232-144	144	zancudo	mosquito	1509	MOSQUITO
+Rowe-1954-232-145	145	comején, hormiga blanca	termite	883	TERMITE
+Rowe-1954-232-146	146	hormiga	ant	587	ANT
+Rowe-1954-232-147	147	araña	spider	843	SPIDER
+Rowe-1954-232-148	148	nigua, pique	chigger	438	CHIGGOE (JIGGER FLEA)
+Rowe-1954-232-149	149	monte	brush	1821	BRUSH
+Rowe-1954-232-150	150	pajonal, sabana	open grassland	260	SAVANNA
+Rowe-1954-232-151	151	cerro, loma	hill	614	HILL
+Rowe-1954-232-152	152	árbol	tree	906	TREE
+Rowe-1954-232-153	153	hoja	leaf	628	LEAF
```

Comment:
LEAF (LEAFLIKE OBJECT). Compare with Huber where we also can see the distinction between hoja and hoja de árbol). hoja de árbol should be mapped to LEAF then.

Reply:
But is that really what the concept refers to? I thought hoja de árbol is a more specific kind of leaf (namely that of a tree compared to all other possible leaves), but hoja by itself still always means the botanical kind, not paper or similar objects. Some other lists map hoja to LEAF as well, so I'd like to wait for someone else to confirm or deny this mapping.